source/query/queries.txt

```diff
@@ -1,25 +1,59 @@
-.. _past-query-view:
-
-=================
-View Past Queries
-=================
+=======================================
+Managing Saved Queries and Aggregations
+=======================================
 
 .. default-domain:: mongodb
 
-Compass automatically stores up to 20 most recent queries for each
-collection. To view these queries, click the history icon in the query bar.
+Saving an Aggregation Pipeline
+------------------------------
 
-.. figure:: /images/compass/query-history-view.png
-   :figwidth: 720 px
-   :alt: Query history view
+.. include:: /includes/fact-save-pipeline.rst
+
+.. _favorite-queries:
+
+Adding a Favorite Query
+-----------------------
+
+.. include:: /includes/fact-favorite-query.rst
+
+Viewing Favorited Queries
+-------------------------
+
+1. Click the :guilabel:`History` button at the top of the 
+   :guilabel:`Documents` tab.
+
+   .. figure:: /images/compass/query-history-view.png
+      :figwidth: 720 px
+      :alt: Query history view
 
-.. seealso::
+#. Click the :guilabel:`Favorites` button in the past queries pane.
 
-   To learn how to write queries to filter your data, see
-   :ref:`query-bar`.
+   .. figure:: /images/compass/query-history-favorite.png
+      :figwidth: 316px
+      :alt: Favorites
+
+   See also : :ref:`Viewing recent query history <recent-queries>`
+
+.. _saved-query-view:
+
+Using the :guilabel:`My Queries` View
```

Comment:
[suggestion]

I think the content in this section should be elevated to the "Viewing Favorite Queries" section because this is the newly preferred method for viewing queries. Then, we can create a sub-section called "Using the :guilabel:`Favorites` Tab" that has the content that used to live here:

https://www.mongodb.com/docs/compass/current/query/favorites/.

So I think ultimately we want something like:

```
Viewing Favorited Queries
------------------------

Starting in  |compass| 1.31, you can view your favorited and saved 
queries and aggregation pipelines on the :guilabel:`My Queries` view.....

Using the :guilabel:`Favorites` Tab
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

You can also view favorite queries from the :guilabel:`Favorites` tab from in the :guilabel:`Documents` view. To open the :guilabel:`Favorites` tab:

1. Click the History button at the top of the Documents view...
2. ....
```